lib/wab/impl/shell.rb

```diff
@@ -54,6 +56,9 @@ def start()
                                          DocumentRoot: @http_dir,
                                          MimeTypes: mime_types)
         server.mount(@pre_path, WAB::Impl::Handler, self)
+        if @export_proxy
+          server.mount('/', WAB::Impl::ExportProxy, @http_dir)
+        end
```

Comment:
consider using `if` modifier here since we do not enforce code-line-length..
```ruby
server.mount('/', WAB::Impl::ExportProxy, @http_dir) if @export_proxy
```
or a shorter version (with relative resolution)
```ruby
server.mount('/', ExportProxy, @http_dir) if @export_proxy
```

Reply:
Good point. It started out with multiple lines. Will shorten.